db/node/src/db.rs

```diff
@@ -151,6 +151,13 @@ impl HoprNodeDb {
         let caches = Arc::new(NodeDbCaches::default());
         caches.invalidate_all();
 
+        // hopr_db_entity::ticket::Entity::update_many()
+        // .col_expr(hopr_db_entity::ticket::Column::State,
+        // sea_query::Expr::value(sea_query::Value::Int(Some(AcknowledgedTicketStatus::Untouched.into()))))
+        // .filter(hopr_db_entity::ticket::Column::State.eq(AcknowledgedTicketStatus::BeingRedeemed.into()))
+        // .exec(&tickets_db)
+        // .await?;
```

Comment:
?

Reply:
This test possibly does not make sense anymore, don't know.